### ISSUE TITLE
Request : URL shorter

### ISSUE THREAD

**Issue by rmc0** [zusam/zusam#29]
For sharring : url shorter can be nice

**Comment by nrobinaubertin** [comment]
I'm not sure. You mean for the public links ?
There are a lot of url shortener that we can selfhost, is it in the scope of Zusam ?

**Comment by rmc0** [comment]
yes it was juste an idea, for sure it's not a prority or real needed

**Comment by Neriderc** [comment]
Perhaps this could be closed if not planned?

**Comment by nrobinaubertin** [comment]
I agree. This can be closed
